### PR TITLE
docs: mgr dashboard

### PR DESCRIPTION
**Pull request by wido**
A few documentation updates for the mgr dashboard

### CODE REVIEW

File: doc/mgr/dashboard.rst [hunk 1]
```diff
@@ -11,5 +11,13 @@ configuration key facility. So we can configure them like::
   ceph config-key put mgr/dashboard/$name/server_port $PORT
 
 where ``$name`` is the ID of the ceph-mgr who is hosting this dashboard web app.
-If they are not configured, the web app will be bound to ``127.0.0.1:7000``.
```
Comment (jcsp) [inline doc/mgr/dashboard.rst]: Do we need to lose the explanation of the default here?

Comment (wido) [inline doc/mgr/dashboard.rst]: Yes, in 015bc3f5c59353e24a367ce3aba7fd62b6f44afc this was reverted and the manager will throw a RunTimeError when no address has been configured

Comment (jcsp) [inline doc/mgr/dashboard.rst]: ah, my mistake

File: doc/mgr/dashboard.rst [hunk 2]
```diff
@@ -11,5 +11,22 @@ configuration key facility. So we can configure them like::
...
+addresses.
+
+In addition, make sure that the *dashboard* module is enabled in the ceph.conf
+configuration file:
+
+  [mgr]
+  mgr_modules = dashboard
+
+Please note that the dashboard will *only* start on the manager which is active
+it that moment. Query the Ceph cluster status to see which manager is active.
```
Comment (tchaikov) [inline doc/mgr/dashboard.rst]: s/it/at/

File: doc/mgr/dashboard.rst [hunk 3]
```diff
@@ -11,5 +11,22 @@ configuration key facility. So we can configure them like::
...
 
 where ``$name`` is the ID of the ceph-mgr who is hosting this dashboard web app.
-If they are not configured, the web app will be bound to ``127.0.0.1:7000``.
 
+These settings can also be configured cluster-wide and not manager specific, eg:
+
+  ceph config-key put mgr/dashboard/server_addr $IP
+  ceph config-key put mgr/dashboard/server_port $PORT
+
+If the port is not configured, the web app will be bind to ``7000``.
```
Comment (tchaikov) [inline doc/mgr/dashboard.rst]: s/bind/bound/

File: src/pybind/mgr/dashboard/README.rst [hunk 4]
```diff
@@ -33,6 +33,13 @@ Add this to your ceph.conf on nodes where you run ceph-mgr:
 
 If you use any other ceph-mgr modules, make sure they're in the list too.
 
+A address where the dashboard will listen on needs to be configured as well, set this to :: to listen on all
```
Comment (tchaikov) [inline src/pybind/mgr/dashboard/README.rst]: s/A/An/,

better off quoting the address, like \`\`::\`\`

File: doc/mgr/dashboard.rst (hunk 3 above)
Comment (wido) [inline doc/mgr/dashboard.rst]: Are you sure it's bound? It's binding to an address, correct?

**Comment by liewegas** [comment]
"the web app will bind" or "the web app will be bound"

File: doc/mgr/dashboard.rst (hunk 3 above)
Comment (tchaikov) [inline doc/mgr/dashboard.rst]: yes, i am sure. please see also https://github.com/ceph/ceph/pull/15920#issuecomment-311154907.

**Comment by wido** [comment]
@liewegas: Thanks! Fixed that, should be good now?

**Pull request merged**